client/securedrop_client/sdk/progress.py

```diff
@@ -0,0 +1,19 @@
+from PyQt5.QtCore import pyqtBoundSignal
+
+
+class ProgressProxy:
+    """
+    Relay the current download progress over to the UI; see
+    the FileDownloadProgressBar widget.
+    """
+
+    def __init__(self, inner: pyqtBoundSignal | None) -> None:
+        self.inner = inner
```

Comment:
If `inner` is None, does that indicate a logical error (eg race condition where we instantiated a progressproxy for a ui element that was killed, or just a bug in our code)? Maybe we can add some error or debug logs to that effect

Reply:
None is totally valid; let's discuss this below where you flagged the usage.